provision/acc_provision/test_main.py

```diff
@@ -786,6 +798,7 @@ def get_args(**overrides):
         "skip_kafka_certs": True,
         "upgrade": False,
         "disable_multus": 'true',
+        "operator_mode": False,
```

Comment:
I think we should put a comment here that at some point we need to make the default as True so that all new test cases are automatically run with this. In fact, it would be good to evaluate how much extra effort is needed to make this default now.

Reply:
Since this operator-mode flag is to be used only by the ansible operator internally, maybe it makes more sense to keep it False by default for our testcases?

I meant to post this reply here

Reply:
Okay for now, lets make a note of this point, and discuss it as a follow up to this patch.